lib/surgex/parser/parsers/contain_parser.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Surgex.Parser.ContainParser do
+  @moduledoc """
+  Checks if the given parameter's value is on the list of allowed values
+  """
+
+  def call(nil, _allowed_values), do: {:error, :value_is_not_allowed}
```

Comment:
It would probably be better to return `{:ok, nil}` like most parsers do for nil input and depend on require parser to catch nil input. This way this parser can work both in cases with and without nil input.